micromamba/tests/test_activation.py

```diff
@@ -279,710 +268,697 @@ def shvar(v, interpreter):
         return f"%{v}%"
 
 
-class TestActivation:
-    @staticmethod
-    def to_dict(out, interpreter="bash"):
-        if interpreter == "cmd.exe":
-            with open(out, "r") as f:
-                out = f.read()
-
-        if interpreter == "fish":
-            return {
-                v.split(" ", maxsplit=1)[0]: v.split(" ", maxsplit=1)[1]
-                for _, _, v in [x.partition("set -gx ") for x in out.splitlines()]
-            }
-        elif interpreter in ["csh", "tcsh"]:
-            res = {}
-            for line in out.splitlines():
-                line = line.removesuffix(";")
-                if line.startswith("set "):
-                    k, v = line.split(" ")[1].split("=")
-                elif line.startswith("setenv "):
-                    _, k, v = line.strip().split(maxsplit=2)
-                res[k] = v
-            return res
-        else:
-            return {k: v for k, _, v in [x.partition("=") for x in out.splitlines()]}
-
-    @pytest.mark.parametrize("interpreter", get_interpreters())
-    def test_shell_init(
-        self,
-        tmp_home,
-        winreg_value,
-        tmp_root_prefix,
-        tmp_path,
-        interpreter,
+def env_to_dict(out, interpreter="bash"):
```

Comment:
All this diff is from making the tests *not* class method.
I think this is better so there is not temptation to use class attributes.